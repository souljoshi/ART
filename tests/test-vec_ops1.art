int main()
{
	vec v1;
	vec v2;
	vec v3;
	vec v4;

	v1 = <66.5, 12.0> + <1.0, 55.1>;
    v2 = <66.5, 12.0> - <1.0, 55.1>;
    v3 = <66.5, 12.0> * <1.0, 55.1>;
    v4 = <66.5, 12.0> / <1.0, 55.1>;

    printf(v1[0]);
    printf(v1[1]);
    printf(v2[0]);
    printf(v2[1]);
    printf(v3[0]);
    printf(v3[1]);
    printf(v4[0]);
    printf(v4[1]);

}