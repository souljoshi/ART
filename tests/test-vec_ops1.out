67.500000
67.100000
65.500000
-43.100000
66.500000
661.200000
66.500000
0.217786
